store/postgres/src/chain_head_listener.rs

```diff
@@ -89,7 +89,7 @@ impl ChainHeadUpdateListener {
                         let logger = logger.clone();
                         let subscribers = subscribers.clone();
 
-                        sender.send(update.clone()).then(move |result| {
+                        tokio::spawn(sender.send(update.clone()).then(move |result| {
```

Comment:
Shall we add the PR messages here as a comment so we know why we're spawning? We may need this again in other places :)